checker/src/main/java/org/checkerframework/checker/guieffect/GuiEffectVisitor.java

```diff
@@ -229,6 +230,17 @@ public Void visitLambdaExpression(LambdaExpressionTree node, Void p) {
         return v;
     }
 
+    @Override
+    protected void checkExtendsImplements(ClassTree classTree) {
+        // Skip this check
```

Comment:
Shouldn't at least the basic `validateTypeOf(im)` checks still be enforced? Like this, one could e.g. write multiple qualifiers on extends/implements.
Having separate methods for the basic and extended validation would allow selective overriding.